### ISSUE TITLE
The file name is too long to delete in windows10

### ISSUE THREAD

**Issue by jiarWang** [facebook/react-native#15602]
'MediaControllerCompat$MediaControllerImp|Api21$ExtraBinderRequestRequestResultReceiver.class', 

I init a project by `create-react-native-app` but it show connect fail when link to https://raw.githubusercontent.com/expo/node-websql/e364fa65146a9e2157a19e5c719e7702c2b6b87a/package.json, however i can see this json file in my ***Chrome***
then I want to deleted this project from my computer, but windows 10 refuse to delete, and show this file's
 name is too long to deal with.

why you give them a such long name.OMG

Wish you can also help me in the reason of connect-fail-problem .

**Comment by hramos** [comment]
Hey, thanks for reporting this issue!

It looks like your description is missing some necessary information, or the list of reproduction steps is not complete. Can you please add all the details specified in the [template](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md)? This is necessary for people to be able to understand and reproduce the issue being reported.

I am going to close this, but feel free to open a new issue with the additional information provided. Thanks!

**Comment by getnum1** [comment]
I think LongPath Tool software can help to fix the issues.

**Comment by Hillery21** [comment]
I agree with getnum1 .. I purchase long path tool and try using it for this problem . It really work for me . Try using it too. :) And let me know how it goes ! I hope it helps!
